### ISSUE TITLE
Semantic segmentation on RSNA dataset

### ISSUE THREAD

**Issue by nguyenpbui** [fuying-wang/MGCA#39]
Hello @fuying-wang ,

Thank you for your code.

I am reproducing your results using the pretrained ResNet50 you provided on the RSNA dataset. When training with 1% training data, I only got 59%, which is lower than the 63% Dice in your paper on the RSNA dataset.
The settings are as follows:
learning rate: 5e-4, batch size: 16, img_size: 512, weight_decay: 1e-6, seed: 42, num_workers: 16.

I ran several times but the results remain the same. Do you have any suggestions for this problem?

Thank you very much.
